CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ----------
 
+[2.1.0] - 2022-10-19
+--------------------
+* **Breaking change**: Removed field ``effort`` from ``CourseCatalogData``
```

Comment:
_If this is a breaking change:_
1. 3.0.0 should be used for breaking changes.
2. You could add more notes to help people understand that this breaking change probably doesn't affect anyone. See 2.0.0 change note.
3. Should use "feat!:" to designate breaking change.
4. Maybe add "optional", as in: "Removed optional field...".

Note: from the standpoint of anyone that was sending effort and now it is no longer appearing, and they still want it, this would be a breaking change. From the perspective of breaking code or services, in theory, removing an optional field is not a breaking change if we have things configured the way we hope, right? So, maybe this doesn't have to be listed as a breaking change. I'll let you decide. However, calling it a breaking change, but not using a breaking change major version bump is unnecessarily inconsistent.

Reply:
This did end up being a breaking change because of how python named arguments work, interestingly. If you call CourseCatalogData(..., effort=5) the code will error. I'll fix the version. This certainly raises some questions for our future work, but for now I'll just call it a breaking change and deal.

Reply:
```TypeError: __init__() got an unexpected keyword argument 'effort'``` was the actual error. I'm actually not sure why we didn't hit this with other fields unless we removed them before we actually tried using them in Studio. Doing a quick code hunt to see.

Reply:
Fixed: updated to 3.0.0 and noted that the field should not actually be in use